docs/src/configuration/app/web/static.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Serve static sites
+sidebarTitle: Static site
+description: Serve completely static sites like in this example.
+---
+
+Static site generators are a popular way to create fast sites.
+Because there's no need to wait for responses from servers, the sites may load faster.
+
+As an example, this documentation is built using a tool called Hugo and served by Platform.sh as a static site.
+You can see the [entire repository on GitHub](https://github.com/platformsh/platformsh-docs),
+including its [app configuration](https://github.com/platformsh/platformsh-docs/blob/main/docs/.platform.app.yaml).
+
+The following example goes through serving a basic static site.
+
```

Comment:
I wouldn't put that in the introduction as it doesn't help clarify the purpose of this guide. I might consider it in a what's next section, but it seems like a separate scenario. It's not necessary for serving static files alongside a dynamic app. That guide is just if you want to change the URL separately from the directories. You can just make your static site output file be `docs` and then it can be served there directly, if you want.

Reply:
Indeed, I misunderstood that one, so safe to ignore the proposal.